docs/ddl-of-carbondata.md

```diff
@@ -1107,6 +1107,12 @@ Users can specify which columns to include and exclude for local dictionary gene
   ALTER TABLE locationTable DROP PARTITION (country = 'US');
   ```
 
+   The below flag is set to enable the feature of moving the deleted partition data to trash when dropped partition command is given.
+   ```
+   carbon.enable.partitiondata.trash
+   ```
```

Comment:
please add in configuration md file also

Reply:
done